src/_guides/technical-writing/5-improvements-to-openapi-operation-documentation.md

```diff
@@ -0,0 +1,255 @@
+    description: Supports order creation and status lookup
+paths:
+  /users:
+    get: ...
+      tags:
+        - User Management
+
+  /users/{id}:
+    get: ...
+      tags:
+        - User Management
+  /orders:
+    get: ...
+      tags:
+        - Order Management
+```
+
+## 5. Improve Consistency in Operation Naming Conventions
+
+Inconsistent naming conventions for operations, parameters, and schema properties can lead to confusion and errors in API integration. Since making changes to these conventions can break your consumers, it is important to take a fresh look at your API operations, parameters, and schema definitions before releasing your API. 
+
+To further improve your API design efforts, involve your technical writers earlier in the API design process. This will provide insights regarding improvements that could be made to improve the developer experience. By shifting left your technical writer involvement in the API lifecycle, you also reduce the cost of change due to late feedback. 
+
+### An Example of Inconsistent or Unclear Naming
+
+Below is an example of API operations that don’t follow a consistent naming convention, resulting in an increased likelihood of mistakes when integrating your API. 
+
+**Example Mistake:**
+```yaml
+paths:
+  /calcSalesTax:
+    post: ...
+  /vatCalculation:
+    post: ...
+components:
+  schemas:
+    CalculationResponse:
+      type: object
+      properties:
+        amount:
+          type: number
+          format: double
+        …
+```
+
+### Improved Version:
+Using a consistent, predictable naming convention improves readability and usability of the API documentation.
+
+**Improved Example:**
+```yaml
+paths:
+  /calculateSalesTax:
+    post: ...
+  /calculateVatTax:
+    post: ...
+components:
+  schemas:
+    CalculatedTax:
+      type: object
+      properties:
+        amount:
+          type: number
+          format: double
+        currency:
+          type: String
+        …
+```
+## Applying OpenAPI Specification Overlays
+
+For teams that are generating OpenAPI Specification documents [directly from their code](https://docs.bump.sh/guides/openapi/code-first/) using annotations, [technical writers can further benefit](https://docs.bump.sh/guides/technical-writing/efficient-tech-writing-process/) from the use of OpenAPI overlays. Refer to [our article that details using overlays](https://docs.bump.sh/guides/openapi/augmenting-generated-openapi/) to improve your tech writing workflow. 
+
+## Final Thoughts on Improving Your API Documentation 
+
+Creating a high-quality OpenAPI Specification v3 document requires attention to detail, thoroughness, and a focus on the user's experience. Take the additional time to review your documentation, look for areas to improve, and identify design flaws early so that they can be corrected with minimal impact to developers. Doing so will produce clear, comprehensive, and usable API documentation that results in a better developer experience. 
```

Comment:
not sure if it's relevant to mention a speficiation (and a version here), what was the intention here?

```suggestion
Creating a high-quality API document requires attention to detail, thoroughness, and a focus on the user's experience. Take the additional time to review your documentation, look for areas to improve, and identify design flaws early so that they can be corrected with minimal impact to developers. Doing so will produce clear, comprehensive, and usable API documentation that results in a better developer experience. 
```